pages/collection/[id].vue

```diff
@@ -400,7 +405,14 @@ try {
     projects = ref(data.data)
 
     creator = ref(auth.value.user)
-    refreshProjects = async () => {}
+    refreshProjects = async () => {
+      const newData = await useAsyncData(`user/${auth.value.user.id}/follows`, () =>
+        useBaseFetch(`user/${auth.value.user.id}/follows`)
+      )
+      Vue.nextTick(() => {
+        projects = ref(newData.data)
+      })
```

Comment:
Understandable, I didn't really understand what `useAsyncData` did, and I think I used next tick because it was not updating until the second click of unfollow (after the data had already changed on the server, but the DOM hadn't refreshed). I also embarrassingly forgot how refs worked as you said. I'll go ahead and make and test the change now.

Reply:
I tried to change it to only use `useBaseFetch`, but it still doesn't update the DOM until a second click even with `nextTick`. A better solution might be to have a boolean ref (i.e. `refProj`) that is watched in `useAsyncData`, similar to in the [Nuxt docs](https://nuxt.com/docs/api/composables/use-async-data#watch-params).
Example:
```typescript

const refProj = ref(false)

async function unfollowProject(project) {
  await userUnfollowProject(project)
  refProj.value = true
  await Vue.nextTick()
  refProj.value = false
}
try {
...
    const data = await useAsyncData(
      `user/${auth.value.user.id}/follows`,
      () => useBaseFetch(`user/${auth.value.user.id}/follows`),
      {
        watch: [refProj],
      }
    )
...
}
```

Reply:
See newest commit for a better version of this solution using `refreshNuxtData`. All problems with needing two clicks are also solved.